whatwg-fetch and breaking boilerplate tests through yarn rw tests

Hey everyone, I see this has come up a few times before, notably in #1264 

I am still getting these errors even after running `yarn rw upgrade` about five minutes ago. 

`yarn rw test` is failing with this error: 

```
  ● Test suite failed to run

    Cannot find module 'whatwg-fetch' from '../node_modules/@redwoodjs/core/dist/configs/browser/jest.setup.js'

      at Resolver.resolveModule (../node_modules/jest-resolve/build/index.js:306:11)
      at Object.<anonymous> (../node_modules/@redwoodjs/core/dist/configs/browser/jest.setup.js:5:1)
```

Looks like I'm running v0.21.0. All of my web tests are failing, including the boilerplate ones from running the page generator like: `yarn rw g page home`. API tests are all fine. 

Are there additional steps I need to take in upgrading, beyond `yarn rw upgrade`? Or is something still broken here? Thank you!

--- a/packages/core/package.json
+++ b/packages/core/package.json
@@ -62,13 +62,13 @@
     "webpack-cli": "^3.3.11",
     "webpack-dev-server": "^3.10.3",
     "webpack-merge": "^5.1.2",
-    "webpack-retry-chunk-load-plugin": "^1.4.0"
+    "webpack-retry-chunk-load-plugin": "^1.4.0",
+    "whatwg-fetch": "^3.5.0"
   },
   "devDependencies": {
     "@types/babel-core": "^6.25.6",
     "@types/babel-plugin-tester": "^9.0.0",
-    "babel-plugin-tester": "^9.0.1",
-    "whatwg-fetch": "^3.0.0"
+    "babel-plugin-tester": "^9.0.1"
   },
   "gitHead": "c235e7d7186e5e258764699c0e0e1d5cc0fdd0b5",
   "scripts": {

